spanner/integration_test.go

```diff
@@ -2043,8 +2043,18 @@ func TestIntegration_BasicTypes_ProtoColumns(t *testing.T) {
 		val  interface{}
 		want interface{}
 	}{
+		// Proto Message
```

Comment:
In addition to adding tests to this giant test, can we create an integration test that could also act like a sample?

So that we can clear see from a users' perspective, how these new interface looks like.

Reply:
@Shua1 
Golang has a separate repository for samples and its associated integration tests that shows how users can use the functionality.
I am working on the samples currently and will be raising a separate PR for those changes.